src/SprykerSdk/Acp/Manifest/Reader/AppManifestReader.php

```diff
@@ -150,7 +156,7 @@ protected function extendCollectionByTranslation(
             ->setTranslations(json_decode((string)file_get_contents($translationFile->getPathname()), true));
 
         if (json_last_error()) {
-            return;
+            throw new InvalidTranslationException(json_last_error_msg());
```

Comment:
Please make the messages more verbose and helpful. I expect something like: `There was an error while decoding the translation file "/path/to/file". Error: ...`

With this, the user would directly see which file is in question and what exactly happened.